libfuse/start.go

```diff
@@ -113,6 +114,13 @@ func Start(options StartOptions, kbCtx libkbfs.Context) *libfs.Error {
 	}
 	defer libkbfs.Shutdown()
 
+	// Report "startup successful" to the supervisor (currently just systemd on
+	// Linux). This isn't necessary for correctness, but it allows commands
+	// like "systemctl start kbfs.service" to report startup errors to the
+	// terminal, by delaying their return until they get this notification
+	// (Type=notify, in systemd lingo).
+	systemd.NotifyStartupFinished()
```

Comment:
Not sure what the semantics are supposed to be, but this notification is obviously sent before we mount KBFS.  Should t go down below, above `mi.Wait()`?  Or is it just supposed to measure the liveness of the process, and not whether it's been fully initialized?

Reply:
Ah, I'll put it as low as possible. I wasn't sure before when that `Wait` call was returning.

Reply:
Hmm, it looks like `startMounting` blocks though.

Reply:
I'm going to leave this as is, I think. We could move it into `startMounting` right before the call to `Serve`. But then I think we'd also need to keep a call here in `Start` for the `SkipMount` case. I'll stick with the simple thing for now, and we can make more than one call if we decide we need to wait for more mount errors.

Reply:
(Also it looks like kbfsfuse doesn't actually exit if the mount fails. That's a slightly weird state to report to systemd. Maybe the right thing to do is exit with a standard error code?)

Reply:
Ah you're right about the blocking, sorry.

There are several schools of thought about what should happen if the mount fails, since we might want to keep running to let chat work.  Still an active topic of debate I think.